requirements-dev.txt

```diff
@@ -0,0 +1,13 @@
+#
```

Comment:
Suggest adding requires.io filters to pin requirements at the current integer (automation and backwards compatibility!)
https://requires.io/features/